mopidy/ext.py

```diff
@@ -198,7 +198,13 @@ def load_extensions():
 
     for entry_point in pkg_resources.iter_entry_points('mopidy.ext'):
         logger.debug('Loading entry point: %s', entry_point)
-        extension_class = entry_point.load(require=False)
+        try:
+            extension_class = entry_point.load(require=False)
+        except Exception as e:
+            logger.exception("Failed to load extension {}: {}".format(
+                entry_point.name, e
+            ), exc_info=e)
```

Comment:
`logger.exception()` will get the `exc_info` itself without explicitly passing it since this is inside an except block.